td/imports/management/commands/language_alt_names.py

```diff
@@ -0,0 +1,86 @@
+"""
+
+USAGE:
+    python manage.py language_alt_names <username> < <file>.csv
+
+"""
+
+import sys
+import traceback
+
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.auth.models import User
+
+from ....models import Language, LanguageAltName
+
+
+class Command(BaseCommand):
+    help = "Import language alternate names from CSV"
+    data = sys.stdin.read()
+    user = None
+    code = None
+    name = None
+
+    def add_arguments(self, parser):
+        parser.add_argument('username', nargs='+', type=str)
+
+    @staticmethod
+    def split_endings(data):
+        """
+        Split string based on CRLF or LF ending
+        :param data: Raw strings that may contain line endings
+        :return: Array of strings
+        """
+        return data.split("\r\n") if "\r\n" in data else data.split("\n")
```

Comment:
I decided to utilize the python `csv` library so this will be taken care of in the next revision.